### PR TITLE
Validation plugin documentation

### PR DESCRIPTION
**Pull request by priyankjalu89**

### CODE REVIEW

**Comment by bvw** [comment]
Those code pens were designed for testing the set form function and to enumerate many variations on checkboxes.
The purpose for this documentation is different and should represent a useful widgets in an example form.

**Comment by bvw** [comment]
* Add notes to each codepen
* Show link to related codepens
* Demo each type of error notification; error, notify, alert

**Comment by priyankjalu89** [comment]
[Validation - Form Save](http://codepen.io/priyank-qcode/pen/pbrmZz)
[Validation - Tooltip for Invalid data](http://codepen.io/priyank-qcode/pen/OXKAKE)

File: docs/validation.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# Validation plugin
+
+This plugin is used with form element to validate user input using ajax.
+
+# Options
```
Comment (PeterChaplin) [inline docs/validation.md]: I'd prefer if we could have only one first-level heading per page, and not jump straight from h1 to h3. What do we want the sections of this page to be, what are the sub-sections of each section, etc?

**Comment by nicky-johnstone** [comment]
I've fleshed out Priyank's draft a bit more and [that copy was merged in](https://github.com/qcode-software/qcode-ui/pull/156) therefore this one is no longer necessary.

**Pull request closed without merging**